lib/tangles/get-tangle-data.js

```diff
@@ -27,6 +27,7 @@ function getTangleRoot(server, tangle, groupId, cb) {
       return cb(new Error(`get-tangle: unknown groupId ${groupId}`))
     }
 
+    // TODO: use getPickedEpoch?
```

Comment:
make a separate issue for this

Reply:
we actually check against the current writeKey here, i missed that when skimming, so this function should be fine actually